contracts/TokenHolderRevenueFund.sol

```diff
@@ -71,10 +83,10 @@ contract TokenHolderRevenueFund is Ownable, AccrualBeneficiary, Servable, Transf
     event CloseAccrualPeriodEvent(int256 periodAmount, int256 aggregateAmount, address currencyCt,
         uint256 currencyId);
     event ClaimAndTransferToBeneficiaryEvent(address wallet, string balanceType, int256 amount,
-        address currencyCt, uint256 currencyId, string standard);
-    event ClaimAndTransferToBeneficiaryByProxyEvent(address wallet, string balanceType, int256 amount,
-        address currencyCt, uint256 currencyId, string standard);
-    event ClaimAndStageEvent(address from, int256 amount, address currencyCt, uint256 currencyId);
+        address currencyCt, uint256 currencyId, uint256 firstPeriodIndex, uint256 lastPeriodIndex,
+        string standard);
+    event ClaimAndStageEvent(address from, int256 amount, address currencyCt, uint256 currencyId,
```

Comment:
Is 'standard' wanted in this event?

Reply:
It is a parameter of the function emitting the event, hence useful to persist for later debugging.